templates/hello.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Hello</title>
+</head>
+<body>
+    <h1>Hello {{ name }}!</h1>
+    <form action="/exit/" method="post">
```

Comment:
И этот атрибут тоже можно убрать
```suggestion
    <form method="post">
```